data-exporter/src/test/java/org/reactome/release/dataexport/UniProtDbIdGenerator.java

```diff
@@ -0,0 +1,9 @@
+package org.reactome.release.dataexport;
+
+public class UniProtDbIdGenerator {
+	private static long uniProtDbId = 1L;
+
+	public static long getNextUniProtDBID() {
```

Comment:
I'm a little confused by what is going on here

Reply:
Before I was using hardcoded numbers to give mock db ids in the tests.  Since the UniProtReactomeEntry objects that get created are cached (so duplicates aren't created), this caused a headache in making sure the numbers didn't collide when running multiple tests.  I created this as a helper class to generate mock db ids starting at 1 and to give the next available number.

Reply:
Fair enough, could a brief comment describing the methods purpose be added?

Reply:
Yup, will do!